src/drivers/sdr/gnuradio.c

```diff
@@ -70,40 +75,48 @@ uint16_t crc16(uint8_t * pData, int length)
 
 //format and send 128B frame to gnuradio via zmq
 int csp_gnuradio_tx(int fd, const void * data, size_t len){
-
-    if(len != PACKET_LEN){
-        while(1); //improve error handling
+    
+    if((int)len != PACKET_LEN){
+        while(1); //todo support len != 128 one day, maybe
```

Comment:
Since this is linux code, should it not exit? Infinite stalls are kind of an embedded thing

Reply:
Or even just return silently

Reply:
Good to know. made it exit